README.md

```diff
@@ -11,15 +11,15 @@ A Document Processing System.
 
 ## Getting Started
 
-TDB
+TBD
 
 ### Prerequisites
 
-TDB
+What things you need to install the software and how to install them
```

Comment:
Can you add a . at the end of the sentence ;-)

Reply:
Period has been added as requested. Please let me know if you 'see' the change I've pushed.